File: DotNetWikipediaTest/DotNetWikipediaTest/PageObjects/HomePage.cs
using System;
using OpenQA.Selenium;

namespace DotNetWikipediaTest.PageObjects
{
    public class HomePage
    {
        private IWebDriver driver;
        private readonly string initialURL = "https://en.wikipedia.org";

        public HomePage(IWebDriver driver)
        {
            this.driver = driver;
        }

        private IWebElement loginPageLink => driver.FindElement(By.Id("pt-login"));

        public void Launch()
        {
            driver.Navigate().GoToUrl(initialURL);
        }

        public LoginPage ClickLoginLink()
        {
            loginPageLink.Click();
            return new LoginPage(driver);
        }
    }
}

File: python-wikipedia-test/pages/home_page.py
import unittest

from .base_page import BasePage
from .login_page import LoginPage
from locators.home_page_locators import HomePageLocators

class HomePage(BasePage):
	def click_login_link(self):
		self.click_button(HomePageLocators.LOGIN_LINK)
		return LoginPage(self.driver)
